app/views/runs/stats/index.slim

```diff
@@ -16,6 +16,7 @@
     ul.run-options
       li
         a href=run_path(@run) &larr; back to run
+      li = link_to 'graphs', run_graphs_path(@run)
```

Comment:
It looks ok to me on the stats page when I offset the table to be in the center, my only concern would be the entire stats page timing out on runs that have long histories.